### ISSUE TITLE
[v2] Generated sitemap is empty

### ISSUE THREAD

**Issue by aeneasr** [facebook/docusaurus#2543]
## 🐛 Bug Report

The file `./build/sitemap.xml`  which is generated by docusaurus-plugin-sitemap is just an empty file.

### Have you read the [Contributing Guidelines on issues](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#reporting-new-issues)?

yes

## To Reproduce

```
$ npx @docusaurus/init@next init my-website classic
$ cd my-website
$ npm run build
$ cat build/sitemap.xml
# file is empty
```

## Expected behavior

Sitemap should be generated.

## Actual Behavior

Sitemap is empty.

## Your Environment

- Docusaurus version used: `npx @docusaurus/init@next`
- Environment name and version (e.g. Chrome 78.0.3904.108, Node.js 10.17.0): v13.11.0
- Operating system and version (desktop or mobile): desktop

## Reproducible Demo

See [To Reproduce](#to-reproduce)

**Comment by yangshun** [comment]
Fixed in #2530

**Comment by aeneasr** [comment]
Oh sorry, I saw that PR but thought it was already released - looks like that's not the case :)
